tests/e2e/deployment-test.sh

```diff
@@ -98,39 +87,62 @@ gcloud container clusters get-credentials "${K8S_CLUSTER}" --region us-central1
 
 # Verify containers running
 echo
-echo -n "Verifying deployment result"
-METRICS_CONTAINER_STATE=0
-LOGS_CONTAINER_STATE=0
-
-for _ in {1..60}
-do
-  if [[ $DEPLOYMENT_TYPE == all ]] || [[ $DEPLOYMENT_TYPE == metrics ]]; then
-    check_container_state "dynatrace-gcp-monitor-metrics"
-    METRICS_CONTAINER_STATE=$?
-  fi
-
-  if [[ $DEPLOYMENT_TYPE == all ]] || [[ $DEPLOYMENT_TYPE == logs ]]; then
-    check_container_state "dynatrace-gcp-monitor-logs"
-    LOGS_CONTAINER_STATE=$?
-  fi
-
-  if [[ ${METRICS_CONTAINER_STATE} == 0 ]] && [[ ${LOGS_CONTAINER_STATE} == 0 ]]; then
-    break
-  fi
-
-  sleep 10
-  echo -n "."
-done
+echo "Verifying deployment result"
+check_deployment_status || exit 1
 
 echo
 kubectl -n dynatrace get pods
 
 generate_load_on_sample_app
 
-if [[ ${METRICS_CONTAINER_STATE} == 0 ]] && [[ ${LOGS_CONTAINER_STATE} == 0 ]]; then
-  echo "Deployment completed successfully"
-  exit 0
-else
-  echo "Deployment failed"
-  exit 1
+
+if [[ $TRAVIS_BRANCH == 'master' ]]; then
+  echo "#####PERFOMANCE TEST#####"
```

Comment:
Only minor comment, for readability. Can you add an extra /n before here? And maybe something similar to "#####PERFOMANCE TEST ENDED#####" at the end. So going through the logs we can spot more easily what's related to the perf test. 

Based on what I saw here, if I'm not mistaken: https://app.travis-ci.com/github/dynatrace-oss/dynatrace-gcp-monitor/jobs/598999742#L567